packages/@react-spectrum/calendar/src/CalendarTableBody.tsx

```diff
@@ -23,7 +23,7 @@ export function CalendarTableBody({state}: CalendarTableBodyProps) {
     <tbody>
       {
         [...new Array(state.weeksInMonth).keys()].map(weekIndex => (
-          <tr key={weekIndex}>
+          <tr key={weekIndex} aria-rowindex={weekIndex + 2}>
```

Comment:
I noticed a weeksInMonth + 1 up above, what's with these +1's and +2's?
https://github.com/adobe-private/react-spectrum-v3/pull/188/files#diff-aa122aaed571a90e88b8a601bcbf7afcR187

Reply:
`aria-rowindex` needs to account for the table's header row. `weeksInMonth` are zero-based, but `aria-rowindex` starts at `1`, so the `aria-rowindex` for each table body row will have to be `weekIndex + 2`.

Reply:
`weeksInMonth + 1` is the `aria-rowcount`.